cli/client.go

```diff
@@ -316,6 +316,30 @@ var clientDealCmd = &cli.Command{
 			ref.TransferType = storagemarket.TTManual
 		}
 
+		// Check if the address is a verified client
+		dcap, err := api.StateVerifiedClientStatus(ctx, a, types.EmptyTSK)
+		if err != nil {
```

Comment:
I believe err will be non-nil (because datacap key will be missing) when client is not verified.